src/common/hooks/useGetPlays.js

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect, useContext } from "react";
+import { SearchContext } from "common/search/search-context";
+import { FetchPlaysFilter } from "common/services/request/query/fetch-plays-filter";
+import { FetchPlaysSimple } from "common/services/request/query/fetch-plays";
+
+import { submit } from "common/services/request";
+
```

Comment:
Provide a description of what is this hook supposed to do.

Reply:
ok.